peer/export-consul.ctmpl

```diff
@@ -36,3 +36,8 @@ export {{.Key | toUpper}}="{{.Value}}"
 export {{.Key | toUpper}}="{{.Value}}"
 {{end -}}
 {{end -}}
+
+# Peer namespace, overwrites values to allow peer to inject values
+{{range ls (printf "services/%s/peer/%s/env_vars" (env "SERVICE_NAME") (env "SERVICE_ENV")) -}}
```

Comment:
We don't put `env` in here anymore.  We have a consul/vault per ENV so there is no reason to put `env` in the path.